src/test/java/org/ays/step_definitions/RegistrationApplicationsDetails.java

```diff
@@ -0,0 +1,44 @@
+package org.ays.step_definitions;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import org.ays.browser.AysPageActions;
+import org.ays.pages.RegistrationApplicationsPOM;
+import org.ays.pages.SuperAdminPOM;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RegistrationApplicationsDetails {
+    private final RegistrationApplicationsPOM registrationApplications = new RegistrationApplicationsPOM();
+    private final AysPageActions pageActions = new AysPageActions();
+    private final SuperAdminPOM superAdminPOM = new SuperAdminPOM();
+
+    @Then("I should see the following column headers")
+    public void iShouldSeeTheFollowingColumnHeaders(List<String> expectedHeaders) {
+        List<WebElement> headerCells = registrationApplications.getRegistrationApplicationsListColumnHeaders();
+        List<String> actualHeaders = new ArrayList<>();
+        for (WebElement headerCell : headerCells) {
+            actualHeaders.add(headerCell.getText());
+        }
+        Assert.assertEquals(expectedHeaders, actualHeaders);
+    }
+
+    @And("Select waiting option from the status menu")
+    public void selectWaitingOptionFromTheStatusMenu() {
+        pageActions.clickMethod(registrationApplications.getStatusOptionsFromFilterDropdownMenu().get(0));
+    }
+
+    @And("Click status bar")
+    public void clickStatusBar() {
+        pageActions.clickMethod(superAdminPOM.getStatusBar());
+    }
+
+    @Then("The review button should not be available")
+    public void theReviewButtonShouldNotBeAvailable() {
+        List<WebElement> actionButtons = registrationApplications.getShowButtonUnderTheActions();
+        Assert.assertTrue(actionButtons.isEmpty(), "Action button should not be available");
+    }
```

Comment:
Burada kontrol action bölümündeki detay ekranının görüntülenmesine göre ayarlanmak istenmiş anladığım kadarıyla. Ancak statüyü değiştirdiğimizde bu bölüm hata verecektir. Bu kısımda tüm statülerde çalışacak şekilde bir geliştirme yapılması daha uygun olabilir.

Reply:
Burda statüsü waiting olanları filitreleyip ondan sonra action altında show butonu olmadığını test ettim. Statüyü değiştirirsek fail olur dediğin kısmı anlamadım, o kısmı açıklar mısın?

Reply:
Şunu kastetmiştim waiting yerine verified ya da rejected seçilirse assertion boş olana göre ayarlandığı için fail verecektir. Ama sanırım siz diğer statüler için farklı senaryolarda test etmeyi düşündünüz. Ben geliştirme tamamlandı diye böyle bir yorum yapmıştım.

Reply:
Evet aslında her biri icin ayrı senaryolar vardi ama tum kayitlar sistemden silindiği icin onları otomasyona katmanın bir anlamı kalmadı. Esasen bu case'i de kaldırmayı düşünüyorum, su an sistemde waiting olarak bir kayıt olduğu icin geciyor ama o da silinirse yine hata verecektir. Yeni bir commitle bir kaç degisiklik daha yapacagım.

Reply:
aslında kaldırmak yerine bulamaması durumunda bir hata mesajı döndürecek şekilde bir düzenleme yapılabilir. İlerde tekrar testini eklemek yerine böyle bir yol izlenebilir belki.